### PR TITLE
readme clean up

### PR DESCRIPTION
**Pull request by brimigs**

### CODE REVIEW

File: scripts/multisig/osmosis/README.md [hunk 1]
```diff
@@ -181,6 +179,25 @@ QUERY='{"price_sources":{}}'
...
 ```
 
+- Verify Rewards Collector Routes are set correctly:
+
+```shell
+QUERY='{"routes":{}}'
+osmosisd query wasm contract-state smart $OSMO_TEST_REWARDS_COLLECTOR "$QUERY" --output json --node=$OSMO_TEST_NODE
+```
+
+- Verify Admin is set correctly:
```
Comment (piobab) [inline scripts/multisig/osmosis/README.md:189]: You can add this command:
```osmosisd query wasm contract osmo1w5rqrdhut890jplmsqnr8gj3uf0wq6lj5rfdnhrtl63lpf6e7v6qalrhhn```
Zrzut ekranu 2023-01-30 o 23 47 08

File: scripts/multisig/osmosis/README.md [hunk 2]
```diff
@@ -181,6 +179,25 @@ QUERY='{"price_sources":{}}'
...
+  _Note: If admin is not set, contracts are immutable_
+
+```shell
+QUERY='{"routes":{}}'
```
Comment (piobab) [inline scripts/multisig/osmosis/README.md:193]: To be removed

File: scripts/multisig/osmosis/README.md [hunk 3]
```diff
@@ -181,6 +179,25 @@ QUERY='{"price_sources":{}}'
...
+osmosisd query wasm contract $OSMO_TEST_REWARDS_COLLECTOR
```
Comment (piobab) [inline scripts/multisig/osmosis/README.md:194]: This is quite important so maybe include such imgs:

Zrzut ekranu 2023-01-30 o 23 21 52

Zrzut ekranu 2023-01-30 o 23 47 08

**Comment by piobab** [comment]
Small comments left. LGTM but better to confirm with multisig owners that they feel the flow :)

**Review by larry0x — APPROVED** [review]
@brimigs I had too many suggestions (mostly just formatting) so I ended up pushing a commit: https://github.com/mars-protocol/red-bank/pull/172/commits/370a32cb23201bad0b85d7cc06e525a79acc6355. (some devs consider it rude that other people push commits to their PRs, so sorry for this...)

For Markdown code blocks, I've found that using ` ```shell ` doesn't highlight the syntax correctly in some cases, so I replaced them all with ` ```bash `.

File: scripts/multisig/osmosis/README.md (hunk 3 above)
Comment (brimigs) [reply scripts/multisig/osmosis/README.md]: added an admin query :)

**Comment by brimigs** [comment]
> @brimigs I had too many suggestions (mostly just formatting) so I ended up pushing a commit: [370a32c](https://github.com/mars-protocol/red-bank/commit/370a32cb23201bad0b85d7cc06e525a79acc6355). (some devs consider it rude that other people push commits to their PRs, so sorry for this...)
> 
> For Markdown code blocks, I've found that using `` ```shell `` doesn't highlight the syntax correctly in some cases, so I replaced them all with `` ```bash ``.

I don't think its rude, I appreciate the feedback :) & good point ```shell`` did show some funky syntax on the readme

**Pull request merged**